### PR TITLE
Behindertenmehrbedarf for ALG 2

### PR DESCRIPTION
**Pull request by TeBackh**
### What problem do you want to solve?

Reference the issue or discussion, if there is any. Provide a description of your
proposed solution.

### Todo

- [ ] Target the right branch and pick an appropriate title.
- [ ] Put `Closes #XXXX` in the first PR comment to auto-close the relevant issue once
  the PR is accepted. This is not applicable if there is no corresponding issue.
- [ ] If it is a relevant contribution, put it in `CHANGES.rst`.
- [ ] Any steps that still need to be done.

### CODE REVIEW

File: gettsim/demographic_vars.py [hunk 1]
```diff
@@ -260,3 +261,9 @@ def jüngstes_kind_oder_mehrling(
     """
     out = (alter_monate - alter_monate_jüngstes_mitglied_hh < 0.1) and kind
     return out
+
+def behindert(
+    behinderungsgrad: int
+    ):
+    out = behinderungsgrad>=30
```
Comment (Eric-Sommer) [inline gettsim/demographic_vars.py:268]: maybe add parmeter for the 30 degrees

**Comment by Eric-Sommer** [comment]
@MaxBlesch and I had some discussions on this. The 'mehrbedarf' for disabled persons can only be applied for disabled persons who are the same time part of the labor force. If they are not, they are treated in SGB XII. In addition, they need to receive 'Leistungen zur Teilhabe am Arbeitsleben' oder 'Eingliederungshilfen'. Those kind of things seem very hard to determine, be it by gettsim or by the gettsim user.
There is hardly any objective information one can use to determine whether a specific person is disabled (i.e. some degree of impairment), but could potentially take up work, or not. I therefore wonder how eligibility tests should be modelled. Thoughts, @ChristianZimpelmann?

**Comment by hmgaudecker** [comment]
Great analysis, thanks!

Sounds almost like we want a separate state variable that would determine which SGB applies below normal retirement age? I assume that currently it is ALG II only?

What are the empirics from the statistics on recipients? 0.1% SGB XII below age 66 or more like 5% of total social assistance recipients?

**Comment by Eric-Sommer** [comment]
> Sounds almost like we want a separate state variable that would determine which SGB applies below normal retirement age? I assume that currently it is ALG II only?

Correct. Below retirement age, there is currently only a distinction between  'employed' and 'not employed' plus 'in education'.

> 
> What are the empirics from the statistics on recipients? 0.1% SGB XII below age 66 or more like 5% of total social assistance recipients?

According to Destatis, there are around 1M recipients for 'Grundsicherung bei Erwerbsminderung' and 'Grundsicherung im Alter'. These are about equally split on those below or above the 'Altersgrenze'. 'Leistungen zur Teilhabe am Arbeitsleben' is received by around 250.000 people. By and large, these are disabled persons employed in specific 'Werkstätten'.

**Comment by hmgaudecker** [comment]
> > What are the empirics from the statistics on recipients? 0.1% SGB XII below age 66 or more like 5% of total social assistance recipients?
> 
> According to Destatis, there are around 1M recipients for 'Grundsicherung bei Erwerbsminderung' and 'Grundsicherung im Alter'. These are about equally split on those below or above the 'Altersgrenze'. 'Leistungen zur Teilhabe am Arbeitsleben' is received by around 250.000 people. By and large, these are disabled persons employed in specific 'Werkstätten'. Those seem to be the ones that can qualify for the 'Mehrbedarf'

Great, thanks. So this is quantitatively important, both for accounting reasons and to get labour supply elasticities right. We'll definitely need this distinction.

**Comment by hmgaudecker** [comment]
Duplicate of #370

**Pull request closed without merging**